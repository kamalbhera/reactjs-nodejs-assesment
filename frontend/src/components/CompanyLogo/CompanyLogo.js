import React from 'react';
import './CompanyLogo.scss';

const CompanyLogo = ({ company, logo }) => {
  return (
    <div className="company-logo">
			<img src={logo} alt={company}/>
		</div>
  );
};

export default CompanyLogo;
