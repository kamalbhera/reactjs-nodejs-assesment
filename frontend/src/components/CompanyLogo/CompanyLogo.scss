.company-logo {
  img {
    width: 5rem
  }
}
