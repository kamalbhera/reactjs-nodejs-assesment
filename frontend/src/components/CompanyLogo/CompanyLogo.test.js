import React from 'react';
import { shallow } from 'enzyme';
import CompanyLogo from './CompanyLogo';

it('renders CompanyLogo component', () => {
	const props = {
		logo: "http://host.com/photo.jpg",
		company:"acme"
	};
	const result = shallow(<CompanyLogo { ...props }/>);
	const html = <div className="company-logo"><img src="http://host.com/photo.jpg" alt="acme"/></div>;
	expect(result).toMatchElement(html);
	expect(result).toMatchSnapshot();
});
