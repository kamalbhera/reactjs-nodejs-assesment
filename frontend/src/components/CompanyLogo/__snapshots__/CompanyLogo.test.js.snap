// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CompanyLogo component 1`] = `
<div
  className="company-logo"
>
  <img
    alt="acme"
    src="http://host.com/photo.jpg"
  />
</div>
`;
